README.md

```diff
@@ -202,38 +292,43 @@ _that_ format, parse and display it in whatever display format you want. It can
 parse user input and _get_ the E.164 format to pass _into_ a schema.
 
 ### PostalCode
-We're going to start with a limited set as suggested [here] (http://www.pixelenvision.com/1708/zip-postal-code-validation-regex-php-code-for-12-countries/)
-and [here] (https://stackoverflow.com/questions/578406/what-is-the-ultimate-postal-code-and-zip-regex).
+
+We're going to start with a limited set as suggested [here](http://www.pixelenvision.com/1708/zip-postal-code-validation-regex-php-code-for-12-countries/)
+and [here](https://stackoverflow.com/questions/578406/what-is-the-ultimate-postal-code-and-zip-regex).
 
 Which gives us the following countries:
 
-- US - United States
-- UK - United Kingdom
-- DE - Germany
-- CA - Canada
-- FR - France
-- IT - Italy
-- AU - Australia
-- NL - Netherlands
-- ES - Spain
-- DK - Denmark
-- SE - Sweden
-- BE - Belgium
-- IN - India
+* US - United States
+* UK - United Kingdom
+* DE - Germany
+* CA - Canada
+* FR - France
+* IT - Italy
+* AU - Australia
+* NL - Netherlands
+* ES - Spain
+* DK - Denmark
+* SE - Sweden
+* BE - Belgium
+* IN - India
 
 This is really a practical decision of weight (of the package) vs. completeness.
 
-In the future we might expand this list and use the more comprehensive list found [here] (http://unicode.org/cldr/trac/browser/tags/release-26-0-1/common/supplemental/postalCodeData.xml).
+In the future we might expand this list and use the more comprehensive list found [here](http://unicode.org/cldr/trac/browser/tags/release-26-0-1/common/supplemental/postalCodeData.xml).
 
+### RegularExpression
 
-## Future
-We'd like to keep growing this package, within reason, to include the scalar types that are widely
-required when defining GraphQL schemas. We welcome both suggestions and pull requests. One idea
-we're considering is:
+A `GraphQLScalarType` object generator that takes two arguments:
 
-- BLOB, could be could be a base64-encoded object of some kind
+* `name` - The name of your custom type
+* `regex` - The regex to be used to check against any values for fields with this new type
+
+```
+const MyRegexType = new RegexType('MyRegexType', /^ABC$/);
```

Comment:
I would recommend thinking about using an object with named params for the function signature instead of 2 params. It provides a simple escape hatch for future modifications to your function signature whilst enabling a simple approach to backwards compatibility.
e.g ->`const MyRegexType = new RegexType({ name: 'MyRegexType', regex: /^ABC$/ });` 

Then again I will admit I am biased 😂